docs/src/start/connect-c8y.md

```diff
@@ -248,6 +248,54 @@ Persisting tedge-agent on reboot.
 tedge-agent service successfully started and enabled!
 ```
 
+If your device does not have an internet access and you want to create configuration for requested bridge, you can run a `tedge connect c8y` with an `--offline` flag. 
+
+```sh
+sudo tedge connect c8y --offline
+```
+
+```text title="Output"
+Checking if systemd is available.
+
+Checking if configuration for requested bridge already exists.
+
+Validating the bridge certificates.
+
+Offline mode. Skipping device creation in Cumulocity cloud.
+
+Saving configuration for requested bridge.
+
+Restarting mosquitto service.
+
+Awaiting mosquitto to start. This may take up to 5 seconds.
+
+Enabling mosquitto service on reboots.
+
+Successfully created bridge connection!
+
+Offline mode. Skipping connection check.
+
+Checking if tedge-mapper is installed.
+
+Starting tedge-mapper-c8y service.
+
+Persisting tedge-mapper-c8y on reboot.
+
+tedge-mapper-c8y service successfully started and enabled!
+
+Enabling software management.
+
+Checking if tedge-agent is installed.
+
+Starting tedge-agent service.
+
+Persisting tedge-agent on reboot.
+
+tedge-agent service successfully started and enabled!
+```
+
+In order for the above command to work, the device must be pre-registered in your tenant. If the internet connection is restored, the thin-edge.io instance will establish a connection with the cloud.
```

Comment:
As there is no attempt to connect the cloud, `tedge connect c8y --offline` should work even if the device is not registered yet.

```suggestion
If the device certificate is trusted by c8y when the internet connection is restored,  the thin-edge.io instance will  automatically establish a connection with the cloud.
```

Reply:
resolved in ad228b8049743cff3f0b8a463392554dceb7fd8c